src/lib/transaction.ts

```diff
@@ -52,6 +64,265 @@ export const getSimulationComputeUnits = async (
     sigVerify: false,
   });
 
-  getErrorFromRPCResponse(rpcResponse);
+  if (rpcResponse.value.err) {
```

Comment:
imo these should be wrapped in try catch block, 

in instances where `rpcResponse.value` is undefined, it will throw an error and break.

for this specific error `rpcResponse?.value?.err` makes sense.

Reply:
That makes sense! Thanks, updated